### PR TITLE
[#174140521] Add logic app module

### PR DESCRIPTION
**Pull request by pasqualedevita**
azurerm version 2.18.0

### CODE REVIEW

File: azurerm_logic_app/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+output "id" {
+  value = azurerm_logic_app_workflow.logic_app.id
+}
+
+output "name" {
+  value = var.name
```
Comment (uolter) [inline azurerm_logic_app/outputs.tf:6]:
```suggestion
  value = azurerm_logic_app_workflow.logic_app.name
```

Comment (uolter) [reply azurerm_logic_app/outputs.tf:6]: The real name of the resource is the one assigned to the resource. In your case resource_name

File: azurerm_logic_app/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+output "resource_name" {
```
Comment (uolter) [inline azurerm_logic_app/outputs.tf:9]: Actually there is no need to expose the resource_name .....

File: azurerm_logic_app/outputs.tf (hunk 1 above)
Comment (pasqualedevita) [reply azurerm_logic_app/outputs.tf]: yes, sorry.. fixed

File: azurerm_logic_app/outputs.tf (hunk 2 above)
Comment (pasqualedevita) [reply azurerm_logic_app/outputs.tf]: ok, removed

**Pull request merged**